satellite/repair/repairer/ec.go

```diff
@@ -117,7 +117,10 @@ func (ec *ECRepairer) Get(ctx context.Context, limits []*pb.AddressedOrderLimit,
 				if err != nil {
 					// gather nodes that failed to match piece hash with the original piece hash
```

Comment:
`// gather nodes that failed to match piece hash with the original piece hash` shouldn't this be `gather nodes where the calculated piece hash doesn't match the uplink signed piece hash`